Book-A-Desk.Api/Program.fs

```diff
@@ -66,6 +67,20 @@ let configureDynamoDB (sp : ServiceProvider) =
     Console.WriteLine(dynamoDBConfiguration.ReservationTableName)
     Console.WriteLine(dynamoDBConfiguration.OfficeTableName)
 
+let configureEmailService (sp : ServiceProvider) =
+    let config = sp.GetService<IConfiguration>()
+    let emailServiceConfiguration =
+        {
+            SmtpClientUrl = config.["SMTP:ClientUrl"]
+            SmtpUsername = config.["SMTP:Username"]
+            SmtpPassword = config.["SMTP:Password"]
+            EmailSender = config.["SMTP:EmailSender"]
+            EmailReviewer = config.["SMTP:EmailReviewer"]
+        } 
+    Console.WriteLine(emailServiceConfiguration.SmtpClientUrl)
```

Comment:
Beyond testing, I don't think it's worth writing these to the console.

By the way, F# has printfn, which is equivalent to Console.Writeline

Reply:
To Do: Refactor using printfn. Nice to have.